corehq/apps/accounting/forms.py

```diff
@@ -1993,77 +1996,39 @@ def send_message(self):
                                     email_from=settings.DEFAULT_FROM_EMAIL)
 
 
-class AnnualPlanContactForm(forms.Form):
-    name = forms.CharField(
-        label=gettext_noop("Name")
-    )
-    company_name = forms.CharField(
-        required=False,
-        label=gettext_noop("Company / Organization")
-    )
-    message = forms.CharField(
-        required=False,
-        label=gettext_noop("Message"),
-        widget=forms.Textarea(attrs={"class": "vertical-resize"})
-    )
+class EnterprisePlanContactForm(PlanContactForm):
+
+    def send_message(self):
+        subject_tag = "[Enterprise Plan Request]"
+        super().send_message(subject_tag)
+
+
+class AnnualPlanContactForm(PlanContactForm):
 
     def __init__(self, domain, web_user, on_annual_plan, data=None, *args, **kwargs):
-        self.domain = domain
-        self.web_user = web_user
-        super(AnnualPlanContactForm, self).__init__(data, *args, **kwargs)
-        from corehq.apps.domain.views.accounting import (
-            DomainSubscriptionView,
-            SelectPlanView,
-        )
-        self.helper = FormHelper()
-        self.helper.label_class = 'col-sm-3 col-md-2'
-        self.helper.field_class = 'col-sm-9 col-md-8 col-lg-6'
-        self.helper.form_class = "form-horizontal"
+        from corehq.apps.domain.views.accounting import DomainSubscriptionView
         if on_annual_plan:
-            back_button_text = "Back to my Subscription"
-            urlname = DomainSubscriptionView.urlname
-        else:
-            back_button_text = "Select different plan"
-            urlname = SelectPlanView.urlname
-        self.helper.layout = crispy.Layout(
-            'name',
-            'company_name',
-            'message',
-            hqcrispy.FormActions(
-                hqcrispy.LinkButton(
-                    _(back_button_text),
-                    reverse(urlname, args=[self.domain]),
-                    css_class="btn btn-default"
-                ),
-                StrictButton(
-                    _("Submit"),
-                    type="submit",
-                    css_class="btn-primary",
-                ),
-            )
-        )
+            kwargs['back_button_text'] = "Back to my Subscription"
+            kwargs['back_button_url'] = DomainSubscriptionView.urlname
```

Comment:
I assume it's not likely to matter, but these will overwrite without warning if either are passed in. What do you think of using `setdefault` instead?
```suggestion
            kwargs.setdefault('back_button_text', "Back to my Subscription")
            kwargs.setdefault('back_button_url', DomainSubscriptionView.urlname)
```